crates/db_schema_file/src/enums.rs

```diff
@@ -196,3 +196,21 @@ pub enum CommunityFollowerState {
   Pending,
   ApprovalRequired,
 }
+
+#[derive(
+  EnumString, Display, Debug, Serialize, Deserialize, Default, Clone, Copy, PartialEq, Eq, Hash,
+)]
+#[cfg_attr(feature = "full", derive(TS, DbEnum))]
+#[cfg_attr(
+  feature = "full",
+  ExistingTypePath = "crate::schema::sql_types::VoteShowEnum"
+)]
+#[cfg_attr(feature = "full", DbValueStyle = "verbatim")]
+#[cfg_attr(feature = "full", ts(export))]
+/// Lets you show votes for others only, show all votes, or hide all votes.
+pub enum VoteShow {
+  #[default]
+  ShowForOthers,
+  Show,
+  Hide,
+}
```

Comment:
I've only added this new `VoteShow` enum to the `show_downvotes` field. I didn't add it to `show_upvotes`, as I can't think of the use case there, but it could easily be done.

The other place it *might* make sense to add it, is to `show_upvote_pct` , but I'm not sure.